user_tools/docs/index.md

```diff
@@ -91,7 +91,8 @@ The following table summarizes the commands supported for each cloud platform:
 |          +---------------+-------------+---------------------------------+----------+
 |          | profiling     | local       |           unsupported           |    N/A   |
 |          |               +-------------+---------------------------------+----------+
-|          |               | serverless  |           unsupported           |    N/A   |
+|          |               | serverless  | spark_rapids_user_tools \       |  23.04+  |
+|          |               |             |   dataproc profiling [ARGS]     |          |
```

Comment:
Shouldn't it be local mode that is supported and not serverless for Dataproc profiling?

Reply:
Fixed!